src/models/PrimerSessionIntent.ts

```diff
@@ -0,0 +1,4 @@
+export enum PrimerSessionIntent {
```

Comment:
`PrimerPaymentMethodIntent` I am using this one. Should we all align? Not sure what's the best option